api/v1alpha/framework/playwright-cucumberjs.schema.json

```diff
@@ -0,0 +1,178 @@
+  "$id": "https://saucelabs.com/schema/v1alpha/playwright-cucumberjs.schema.json",
+  "title": "saucectl playwright cucumberjs runner configuration",
+  "description": "Configuration file for cucumber using saucectl",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "../subschema/artifacts.schema.json"
+    },
+    {
+      "$ref": "../subschema/docker.schema.json"
+    },
+    {
+      "$ref": "../subschema/npm.schema.json"
+    },
+    {
+      "$ref": "../subschema/sauce.schema.json"
+    },
+    {
+      "$ref": "../subschema/reporters.schema.json"
+    }
+  ],
+  "properties": {
+    "apiVersion": {
+      "const": "v1alpha"
+    },
+    "kind": {
+      "const": "playwright-cucumberjs"
+    },
+    "showConsoleLog": {
+      "$ref": "../subschema/common.schema.json#/definitions/showConsoleLog"
+    },
+    "defaults": {
+      "description": "Settings that are applied onto every suite by default, if no value is set on a suite explicitly.",
+      "type": "object",
+      "properties": {
+        "mode": {
+          "$ref": "../subschema/common.schema.json#/definitions/mode"
+        },
+        "timeout": {
+          "$ref": "../subschema/common.schema.json#/definitions/timeout"
+        }
+      },
+      "additionalProperties": false
+    },
+    "rootDir": {
+      "$ref": "../subschema/common.schema.json#/definitions/rootDir"
+    },
+    "playwright": {
+      "description": "Contains details specific to the playwright.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "$ref": "../subschema/common.schema.json#/definitions/version"
+        }
+      },
+      "required": [
+        "version"
+      ],
+      "additionalProperties": false
+    },
+    "env": {
+      "$ref": "../subschema/common.schema.json#/definitions/env"
+    },
+    "suites": {
+      "description": "The set of properties providing details about the test suites to run.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The name of the test suite, which will be reflected in the test results in Sauce Labs.",
+            "type": "string"
+          },
+          "browserName": {
+            "$ref": "../subschema/common.schema.json#/definitions/browser"
+          },
+          "browserVersion": {
+            "$ref": "../subschema/common.schema.json#/definitions/browserVersion"
+          },
+          "platformName": {
+            "$ref": "../subschema/common.schema.json#/definitions/platformName",
+            "enum": [
+              "macOS 11.00",
+              "macOS 12",
+              "Windows 10",
+              "Windows 11"
+            ]
+          },
+          "screenResolution": {
+            "$ref": "../subschema/common.schema.json#/definitions/screenResolution"
+          },
+					"env": {
+						"$ref": "../subschema/common.schema.json#/definitions/env"
+					},
```

Comment:
Can you make sure that the formatting here is correct?

Reply:
ah.. let me update them